core/services/ocr2/plugins/generic/telemetry_adapter.go

```diff
@@ -49,7 +52,7 @@ func (t *TelemetryAdapter) getOrCreateEndpoint(contractID string, telemetryType
 	key := [4]string{network, chainID, contractID, telemetryType}
 	e, ok := t.endpoints[key]
 	if !ok {
-		e = t.endpointGenerator.GenMonitoringEndpoint(network, chainID, contractID, telemetryType)
+		e = t.endpointGenerator.GenMonitoringEndpoint(contractID, synchronization.TelemetryType(telemetryType), network, chainID)
```

Comment:
Is this order important? I find the old way more intuitive :shrug:

Reply:
I agree: that is the correct ordering for the functions though. I can follow up with a refactor of the underlying types to bring this in line though.